src/main/java/com/github/spring/jwt/JwtCrackerApplication.java

```diff
@@ -75,4 +75,8 @@ public static void main(String[] args) throws Exception {
         SpringApplication.run(JwtCrackerApplication.class, args).close();
         System.exit(0);
     }
+    
+    private void unusedmethod() {

```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "unusedmethod" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUnusedPrivateMethod)